community/security/src/main/java/org/neo4j/server/security/auth/Credential.java

```diff
@@ -83,13 +84,21 @@ private boolean byteEquals( byte[] actual, byte[] given )
             return false;
         }
         boolean result = true;
+        boolean accessible = true;
```

Comment:
Maybe rename this to smth like `actualArrayIsNotEmpty`? Otherwise `accessible` does not make sense for byte arrays.

Reply:
This is to handle to accessible case as explained in the comment.